packages/core/src/http-client/csrf-token-header.ts

```diff
@@ -58,7 +62,10 @@ function makeCsrfRequest<T extends HttpRequestConfig>(
   return executeHttpRequest(destination, appendSlash(axiosConfig))
     .then(response => response.headers)
     .catch(e1 => {
-      logger.error(
+      if (hasCsrfToken(e1)) {
```

Comment:
Perhaps give a comment or a documentation on the `hasCsrfToken` method that the tkoen can be present even if the request has failed.